src/map/clif.c

```diff
@@ -20377,6 +20381,19 @@ static void clif_monster_hp_bar(struct mob_data *md, struct map_session_data *sd
 #endif
 }
 
+static bool clif_show_monster_hp_bar(const struct mob_data *md, struct block_list *bl) {
+	
```

Comment:
nullpo check is missing for the arguments

Reply:
Please see changes in last commit